src/utils/misc.ts

```diff
@@ -133,12 +138,20 @@ export const encrypt: (
   encryptionKey: string,
 ) => Promise<{ iv: string; content: string } | undefined> = async (text, encryptionKey) => {
   try {
+    // create IV as a Buffer
     const iv = randomBytes(IV_LENGTH);
-    const cipher = createCipheriv(DACO_ENCRYPTION_ALGO, encryptionKey, iv);
+    const cipher = createCipheriv(
+      DACO_ENCRYPTION_ALGO,
+      Buffer.from(encryptionKey, CHAR_ENCODING),
+      iv,
+    );
     const encrypted = Buffer.concat([cipher.update(text), cipher.final()]);
+    // split into 64-character lines, so -A is not needed in openssl command, apparently can be buggy with longer files
+    // https://wiki.openssl.org/index.php/Command_Line_Utilities#Base64_Encoding_Strings
+    const encodedContent = encrypted.toString(EMAIL_CONTENT_ENCODING).replace(/(.{64})/g, '$1\n');
     return {
       iv: iv.toString(CHAR_ENCODING),
-      content: encrypted.toString(CHAR_ENCODING),
+      content: encodedContent,
     };
   } catch (err) {
     console.warn('Encryption failure: ', err);
```

Comment:
this should probably be an error message not warning, I missed it in a previous PR

Reply:
the caller of this function here: https://github.dev/icgc-argo/dac-api/blob/develop/src/routes/applications.ts#L282  needs to return 500 if it fails to send the email

as future enhancment it will be good to also notify us somehow since this is an important step, we can create a ticket for that